### ISSUE TITLE
[MU4 Task] [Maybe] Rename CMake project and executable file from `mscore` to `MuseScore`

### ISSUE THREAD

**Issue by cbjeukendrup** [musescore/MuseScore#13259]
**Task description**
The name of the CMake project and the name of the created executable file is still `mscore`. It would be nice to rename this to just `MuseScore`. 

If we indeed want to do this, we should do it before the release of 4.0, because it _might_ be a breaking change (as in: a significant difference from MU3) for Linux users. (On Windows, the name "mscore" is not used anyway, and on macOS, the change won't affect the normal use case, only running from command line might be affected.)

**Comment by shoogle** [comment]
I agree it is strange to call the project `mscore`. It would be better to call it `MuseScore`, though I think the binary should still be called `mscore` on Linux, or at least `musescore` (lowercase). Setting [OUTPUT_NAME](https://cmake.org/cmake/help/latest/prop_tgt/OUTPUT_NAME.html) or [RUNTIME_OUTPUT_NAME](https://cmake.org/cmake/help/latest/prop_tgt/RUNTIME_OUTPUT_NAME.html) to `mscore` or `musescore` might do the trick.

**Comment by HemantAntony** [comment]
I 100% agree. It's something that's always bothered me

**Comment by Wim4rk** [comment]
In linux terminal, musescore3 is the command to open just that. But 5 times out of 10 i only type musescore. mscore is fine by me, but I can create aliases for any command.

**Comment by cbjeukendrup** [comment]
Just found out we need to be careful when changing the executable name, in order not to break uhm... Muse Sounds. https://github.com/orgs/musescore/discussions/17801#discussioncomment-9694585
